pubnub-gson/pubnub-gson-api/src/main/java/com/pubnub/api/java/models/consumer/objects_api/channel/PNRemoveChannelMetadataResult.java

```diff
@@ -1,7 +1,7 @@
-package com.pubnub.api.models.consumer.objects_api.channel;
+package com.pubnub.api.java.models.consumer.objects_api.channel;
 
 import com.google.gson.JsonElement;
-import com.pubnub.api.models.consumer.objects_api.EntityEnvelope;
+import com.pubnub.api.java.models.consumer.objects_api.EntityEnvelope;
 import lombok.AccessLevel;
 import lombok.Getter;
 import lombok.Setter;
```

Comment:
hmm I think I'd rather do it in the future, it seems like they differ by one field, but also they have different super classes so it might be a larger change